index.js

```diff
@@ -18,13 +18,13 @@ info(`Org: ${repoOrg}`);
 info(`Repo: ${repoName}`);
 const ref = context.ref;
 
-const branch = () => {
+const getBranch = () => {
```

Comment:
Renamed these so the intent is a bit clearer. I think if you were doing `if (getTag) {` it would have been more obvious there was a bug here